test/fastboot-test.js

```diff
@@ -101,16 +101,15 @@ describe("FastBoot", function() {
       });
   });
 
-  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function() {
+  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function(done) {
     var fastboot = new FastBoot({
-      distPath: fixture('basic-app')
+      distPath: fixture('long-resolving-app')
     });
 
-    return fastboot.visit('/', {
-      destroyAppInstanceInMs: 5
-    })
+    fastboot.visit('/', { destroyAppInstanceInMs: 5 })
       .catch((e) => {
-        expect(e.message).to.equal('App instance was forcefully destroyed in 5ms');
+        expect(e.message).to.equal('Fastboot forcefully destroyed App instance in 5ms');
```

Comment:
@kratiahuja because of this https://github.com/ember-fastboot/fastboot/pull/150#issue-233426083 , the whole point of this PR was not to that, because of mentioned problems :)